tests/phpunit/tests/functions/__wpCheckAlternateFileNames.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * @group functions.php
+ * @covers ::_wp_check_existing_file_names
```

Comment:
```suggestion
 * @covers ::_wp_check_alternate_file_names
```
- Corrected `@covers` annotation